lblod/frontend-reglementaire-bijlage
# Update `@lblod/ember-rdfa-editor` to 6.3.0 and `@lblod/ember-rdfa-editor-lblod-plugins` to 15.0.0

## Overview
This PR includes the following package updates:
- `@lblod/ember-rdfa-editor` to 6.3.0
- `@lblod/ember-rdfa-editor-lblod-plugins` to 15.0.0
    * This PR also includes an update to the editor configuration in order to conform to the new insert-variable API

### How to test/reproduce
- Start the application
- Ensure that insert variables still works in snippets and regulatory statements

### Checks PR readiness
